tiledb/sm/serialization/array.cc

```diff
@@ -588,6 +588,180 @@ Status array_open_deserialize(
   return Status::Ok();
 }
 
+Status maximum_tile_size_to_capnp(
+    uint64_t maximum_tile_size,
+    capnp::MaxTileSize::Builder* max_tile_size_builder) {
+  max_tile_size_builder->setMaxInMemoryTileSize(maximum_tile_size);
+  return Status::Ok();
+}
+
```

Comment:
Is this a duplicate of maximum_tile_size_serialize?

Reply:
appears so, removed.